scripts/generate-crl-testing-certs.sh

```diff
@@ -146,8 +146,8 @@ basicConstraints       = critical,CA:TRUE,pathlen:0
 keyUsage               = critical,keyCertSign,cRLSign
 subjectKeyIdentifier   = hash
 authorityKeyIdentifier = keyid:always,issuer:always
-crlDistributionPoints  = URI:http://localhost:10086/intermediate.crl
-authorityInfoAccess    = OCSP;URI:http://localhost.test/ocsp
+crlDistributionPoints  = URI:http://yourhost:10086/intermediate.crl
```

Comment:
hi @junczhu , can you clarify why localhost is changed to to yourhost?

Reply:
this is to align the test environment of CLI and K8s, crl server at `localhost` can be access from CLI however it would be hard to access from inside of a K8s pod. By using `yourhost` we can directly add a new hostname for the K8s pod rather than update its localhost.

The CRLDistributionPoints is generated in the cert generating script here so I made this update.
* There is no behavior change from CLI test case.